Java/Trees/Validate_binary_Search_Tree.java

```diff
@@ -0,0 +1,96 @@
+import java.io.*;
+ 
+class GFG {
```

Comment:
This clearly shows the code has been plagarised from: https://www.geeksforgeeks.org/a-program-to-check-if-a-binary-tree-is-bst-or-not/

Reply:
Yes I have referred GFG if needed class name can be changed.

Reply:
@mohitsingh1011 I think you are not aware that plagarism in any Open-source event can lead to deduction of points and repeated violation can lead to disqualification.

Reply:
There is a difference between referring and direct copy-pasting